scripts/script_utils.py

```diff
@@ -16,15 +16,34 @@
 
 from __future__ import print_function
 
+import json
 import os
+import re
+import sys
 import subprocess
 
+# Mimic six here, but avoid any dependencies outside
+# the standard library.
```

Comment:
I think all of these CI hacks should go in a PyPI package so

- Other people can benefit from the work
- The quality of that code can be scrutinized somewhere else
- Dependencies are no big deal